### PR TITLE
Corrected tag format to be valid and fixed marketing repo version in …

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
…rollback pipelines

@edx/pipeline-team Please review

### CODE REVIEW

File: edxpipelines/pipelines/deploy_marketing_site.py [hunk 1]
```diff
@@ -87,7 +87,8 @@ def install_pipeline(save_config_locally, dry_run, variable_files, cmd_line_vars
                 # Writing dates to a file should help with any issues dealing with a job
                 # taking place over two days (23:59:59 -> 00:00:00). Only the day can be
                 # affected since we don't use minutes or seconds.
-                'echo -n "release-$(date +%Y-%m-%d-%H:%M)" > ../target/{new_tag}.txt && '
+                # NOTE: Uses UTC
+                'echo -n "release-$(date +%Y-%m-%d-%H.%M)" > ../target/{new_tag}.txt && '
```
Comment (macdiesel) [inline edxpipelines/pipelines/deploy_marketing_site.py]: not to judge, but hours . minutes not hours : minutes?

**Comment by macdiesel** [comment]
👍 if your format is correct.

File: edxpipelines/pipelines/deploy_marketing_site.py (hunk 1 above)
Comment (Dillon-Dumesnil) [inline edxpipelines/pipelines/deploy_marketing_site.py]: @macdiesel Hours:minutes was the issue. Tag names are not allowed to have ASCII control characters. That's why I switched to hours.minutes. It looked weird with dashes. Too jumbled with the rest of the date.

**Comment by feanil** [comment]
:+1:

**Pull request merged**